Index: iMage/jmjrst.main/src/main/java/org/jis/generator/Generator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- iMage/jmjrst.main/src/main/java/org/jis/generator/Generator.java	(revision c069d3841ab924a6a353597f4b4705c2c2ff9172)
+++ iMage/jmjrst.main/src/main/java/org/jis/generator/Generator.java	(date 1588734066945)
@@ -15,20 +15,24 @@
  */
 package org.jis.generator;
 
-import java.awt.Color;
-import java.awt.Font;
-import java.awt.Graphics2D;
-import java.awt.Image;
-import java.awt.RenderingHints;
+import org.jis.Main;
+import org.jis.options.Options;
+
+import javax.imageio.*;
+import javax.imageio.metadata.IIOMetadata;
+import javax.imageio.metadata.IIOMetadataController;
+import javax.imageio.plugins.jpeg.JPEGImageWriteParam;
+import javax.imageio.stream.ImageInputStream;
+import javax.imageio.stream.ImageOutputStream;
+import javax.imageio.stream.MemoryCacheImageOutputStream;
+import javax.swing.*;
+import javax.swing.filechooser.FileFilter;
+import javax.swing.filechooser.FileSystemView;
+import java.awt.*;
 import java.awt.geom.AffineTransform;
 import java.awt.image.AffineTransformOp;
 import java.awt.image.BufferedImage;
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.IOException;
+import java.io.*;
 import java.util.Iterator;
 import java.util.Locale;
 import java.util.Vector;
@@ -36,28 +40,6 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
 
-import javax.imageio.IIOImage;
-import javax.imageio.ImageIO;
-import javax.imageio.ImageReadParam;
-import javax.imageio.ImageReader;
-import javax.imageio.ImageTypeSpecifier;
-import javax.imageio.ImageWriteParam;
-import javax.imageio.ImageWriter;
-import javax.imageio.metadata.IIOMetadata;
-import javax.imageio.metadata.IIOMetadataController;
-import javax.imageio.plugins.jpeg.JPEGImageWriteParam;
-import javax.imageio.stream.ImageInputStream;
-import javax.imageio.stream.ImageOutputStream;
-import javax.imageio.stream.MemoryCacheImageOutputStream;
-import javax.swing.JFileChooser;
-import javax.swing.JOptionPane;
-import javax.swing.ProgressMonitor;
-import javax.swing.filechooser.FileFilter;
-import javax.swing.filechooser.FileSystemView;
-
-import org.jis.Main;
-import org.jis.options.Options;
-
 /**
  * @author <a href="http://www.jgeppert.com">Johannes Geppert</a>
  *
@@ -637,11 +619,11 @@
       // get width and height of the origianl image
       int w = i.getWidth(null);
       int h = i.getHeight(null);
-      rotateImage(i, 90);
+      rotateImage(i, Math.toRadians(90));
       System.out.println("Width: " + w + " Height :" + h);
 
       System.out.println("Drehe Bild:" + file.getAbsolutePath());
-      i = rotateImage(i, angel);
+      i = rotateImage(i, Math.toRadians(angel));
 
       FileOutputStream fos = new FileOutputStream(file);
 
@@ -652,7 +634,7 @@
       iwparam.setCompressionMode(ImageWriteParam.MODE_COPY_FROM_METADATA);
 
       // set JPEG Quality
-      iwparam.setCompressionQuality(0.92f);
+      //iwparam.setCompressionQuality(0.92f);
       writer.write(imeta, new IIOImage(i, null, null), iwparam);
       ios.flush();
       writer.dispose();
